src/components/instant-apps-header/instant-apps-header.tsx

```diff
@@ -35,6 +37,8 @@ const CSS = {
   shadow: true,
 })
 export class InstantAppsHeader {
+  private _sanitizer = new Sanitizer();
```

Comment:
Do you think this Sanitizer should use the whitelist config?

Reply:
Good idea! Done.